examples/udp-client/udp_client.bal

```diff
@@ -2,38 +2,36 @@
 import ballerina/io;
 import ballerina/udp;
 
-public function main() {
+public function main() returns error? {
   
-    // Create a new connectionless udp client.
-    // Optionally, you can provide interface that the socket need to bind 
-    // and the timeout in milliseconds which specifies the read timeout value
-    // udp:Client client = new ("localhost", 2000);
-    udp:Client socketClient = checkpanic new ;
+    // Create a new connectionless UDP client.
+    // Optionally, you can provide the interface that the socket needs to bind 
```

Comment:
Wouldn't it be more correct to say, "provide the address that the socket needs to bind"? And also, shall we make providing the address example line the default, since, in most of the cases, you would actually give the host where you need to connect. So that's the typical case.

Reply:
-1, the client doesn't take a port number like listener. the client automatically picks an ephemeral port from the local machine. if the client machine has multiple network interface we can provide the interface which it needs to bind.
+1 for changing example - `new (localHost = "localhost");`

Reply:
First of all, my bad, I thought the "localhost" here was the remote host, in that case, the earlier code is more suitable for the default case. 

And yes, the first point was simply on the term "interface", where if it's more suitable to say "host" or "address". Generally the interface is something that is identified with names such as "eth0" and so on. In general terms, we bind to an "address".

Reply:
Upated as ```// Create a new connectionless UDP client.
    // Optionally, you can provide the address that the socket needs to bind 
    // and the timeout in milliseconds, which specifies the read timeout value.
    // udp:Client client = new (localHost = "localhost", timeoutInMillis = 2000);
    udp:Client socketClient = check new;```